gamestonk_terminal/common/technical_analysis/volatility_view.py

```diff
@@ -83,3 +83,72 @@ def view_bbands(
         "bbands",
         df_ta,
     )
+
+
+def view_donchian(
+    ticker: str,
+    s_interval: str,
+    df_stock: pd.DataFrame,
+    upper_length: int,
+    lower_length: int,
+    export: str,
+):
+    """Show donchian channels
+
+    Parameters
+    ----------
+    ticker : str
+        Ticker
+    s_interval : str
+        Interval of stock data
+    df_stock : pd.DataFrame
+        Dataframe of stock prices
+    upper_length : int
+        Length of window to calculate upper channel
+    lower_length : int
+        Length of window to calculate lower channel
+    export : str
+        Format of export file
+    """
+    df_ta = volatility_model.donchian(s_interval, df_stock, upper_length, lower_length)
+
+    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
+    if s_interval == "1440min":
+        ax.plot(df_stock.index, df_stock["Adj Close"].values, color="k", lw=3)
+    else:
+        ax.plot(df_stock.index, df_stock["Close"].values, color="k", lw=3)
+    ax.plot(df_ta.index, df_ta.iloc[:, 0].values, "b", lw=1.5, label="upper")
+    ax.plot(df_ta.index, df_ta.iloc[:, 1].values, "b", lw=1.5, ls="--")
+    ax.plot(df_ta.index, df_ta.iloc[:, 2].values, "b", lw=1.5, label="lower")
+    ax.set_title(f"{ticker} donchian")
+    ax.set_xlim(df_stock.index[0], df_stock.index[-1])
+    ax.set_xlabel("Time")
+    ax.set_ylabel("Share Price ($)")
```

Comment:
I think we want to just leave ylabels as 'Price' since this should be compatible with other stuff, which may not be in $